integration_tests/cli/upgrade-demo.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# Copyright 2020 DSR Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+source integration_tests/cli/common.sh
+
+upgrade_height=$RANDOM
+random_string upgrade_info
+
+echo "propose and approve upgrade"
+random_string upgrade_name
+propose=$(dcld tx dclupgrade propose-upgrade --name=$upgrade_name --upgrade-height=$upgrade_height --upgrade-info=$upgrade_info --from alice --yes)
+approve=$(dcld tx dclupgrade approve-upgrade --name=$upgrade_name --from bob --yes)
+proposed_dclupgrade_query=$(dcld query dclupgrade proposed-upgrade --name=$upgrade_name)
+approved_dclupgrade_query=$(dcld query dclupgrade approved-upgrade --name=$upgrade_name)
+plan_query=$(dcld query upgrade plan)
+check_response "$proposed_dclupgrade_query" "Not Found" raw
+check_response "$approved_dclupgrade_query" "\"name\":\"$upgrade_name\"" raw
```

Comment:
In positive checks in CLI integration tests we use the default output mode (which is json). So `raw` should be removed from positive checks. But a space character should be added after the colon for matching to succeed in json mode.

Reply:
removed